src/content/docs/browser/new-relic-browser/browser-pro-features/browser-data-distributed-tracing.mdx

```diff
@@ -61,6 +57,16 @@ To enable distributed tracing for browser monitoring:
 4. Optional: If all of your services are configured to use the w3c trace context headers, you can choose to exclude the `newrelic`, `traceparent`, and `tracestate` headers from requests.
 5. Optional: [Enable cross-origin resource sharing](#cors).
 6. [Redeploy the browser monitoring agent](/docs/browser/new-relic-browser/installation/upgrade-browser-agent) (either restarting the associated APM agent or updating the copy/paste browser installation).
+7. If you have some apps or services that are downstream from your browser app that use the advanced option Infinite Tracing, see these [setup steps](#infinite-tracing).
+
+## Enable Infinite Tracing [#infinite-tracing]
+
+If your browser apps have some downstream services that use the advanced type of distributed tracing called [Infinite Tracing](/docs/distributed-tracing/infinite-tracing/introduction-infinite-tracing), you should enable this feature for your browser apps. This ensures that your root span (the initiating browser app) is included in the rest of the spans tracked by Infinite Tracing.
+
+To set up Infinite Tracing:
+
+1. Complete the steps above to enable [distributed tracing](#enable).
+2. Go to our [Infinite Tracing documentation](/docs/distributed-tracing/infinite-tracing/set-trace-observer) for steps to create a trace observer and to select which apps (data sources) should send trace spans to the Infinite Tracing trace observer.
```

Comment:
I'd use a comma after "trace observer".

Reply:
@urbiz-nr, since that group of words after the "and" doesn't have a subject, I'm inclined to skip the comma.